src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall(String color, int range) {

```

Comment:
The `getRandomBall` method should not accept `color` and `range` as parameters. Instead, it should internally generate a random color using the `ColorSupplier` class and use a predefined constant for the range. This change is necessary to adhere to the task requirements.